src/models/Expense.model.js

```diff
@@ -1,9 +1,35 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.INTEGER,
```

Comment:
Data Type for Amount: The `amount` field is currently defined as `DataTypes.INTEGER`. If it should support decimal values, consider using `DataTypes.FLOAT` or `DataTypes.DECIMAL`.